src/test/resources/movies.json

```diff
@@ -1,135 +1 @@
: "en", "genres": ["Drama", "Action", "Thriller"]}, {"id": 290859, "title": "Terminator: Dark Fate", "poster": "https://image.tmdb.org/t/p/original/vqzNJRH4YyquRiWxCCOH0aXggHI.jpg", "overview": "Decades after Sarah Connor prevented Judgment Day, a lethal new Terminator is sent to eliminate the future leader of the resistance. In a fight to save mankind, battle-hardened Sarah Connor teams up with an unexpected ally and an enhanced super soldier to stop the deadliest Terminator yet.", "release_date": "2019-10-23", "language": "en", "genres": ["Action", "Adventure", "Science Fiction"]}, {"id": 299536, "title": "Avengers: Infinity War", "poster": "https://image.tmdb.org/t/p/original/7WsyChQLEftFiDOVTGkv3hFpyyt.jpg", "overview": "As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain.", "release_date": "2018-04-25", "language": "en", "genres": ["Adventure", "Action", "Science Fiction"]}, {"id": 141052, "title": "Justice League", "poster": "https://image.tmdb.org/t/p/original/eifGNCSDuxJeS1loAXil5bIGgvC.jpg", "overview": "Fuelled by his restored faith in humanity and inspired by Superman's selfless act, Bruce Wayne and Diana Prince assemble a team of metahumans consisting of Barry Allen, Arthur Curry and Victor Stone to face the catastrophic threat of Steppenwolf and the Parademons who are on the hunt for three Mother Boxes on Earth.", "release_date": "2017-11-15", "language": "en", "genres": ["Action", "Adventure", "Fantasy", "Science Fiction"]}, {"id": 671, "title": "Harry Potter and the Philosopher's Stone", "poster": "https://image.tmdb.org/t/p/original/wuMc08IPKEatf9rnMNXvIDxqP4W.jpg", "overview": "Harry Potter has lived under the stairs at his aunt and uncle's house his whole life. But on his 11th birthday, he learns he's a powerful wizard -- with a place waiting for him at the Hogwarts School of Witchcraft and Wizardry. As he learns to harness his newfound powers with the help of the school's kindly headmaster, Harry uncovers the truth about his parents' deaths -- and about the villain who's to blame.", "release_date": "2001-11-16", "language": "en", "genres": ["Adventure", "Fantasy", "Family"]}, {"id": 509585, "title": "7500", "poster": "https://image.tmdb.org/t/p/original/hL2uecLh2rTTbuVbOriXP0PhqIJ.jpg", "overview": "When terrorists try to seize control of a Berlin-Paris flight, a soft-spoken young American co-pilot struggles to save the lives of the passengers and crew while forging a surprising connection with one of the hijackers.", "release_date": "2019-12-26", "language": "en", "genres": ["Thriller", "Drama"]}]
```

Comment:
Why doing a single line? It was readable before and more maintainable if we have to improve the tests so the dataset in the future

Reply:
Just made it readable again ;)

Reply:
Awesome!